src/profile-logic/js-tracer.js

```diff
@@ -353,6 +353,6 @@ export function getJsTracerLeafTiming(
 
   // Return the list of events, sorted alphabetically.
   return [...jsTracerTimingMap.values()].sort(
-    (a, b) => (a.name > b.name ? 1 : -1)
+    (a, b) => (a.name === b.name ? 0 : a.name > b.name ? 1 : -1)
```

Comment:
Please no nested ternaries, they are very hard to read and reason with.

Reply:
The style that prettier uses doesn't make it easy to understand either.